### PR TITLE
DISTMYSQL-121 Release notes for the PDPXC 8.0.23 update

### PR DESCRIPTION
**Pull request by nastena1606**
Includes HAProxy 2.3.14
modified:   source/conf.py
	new file:   source/release-notes-pxc-v8.0.23.upd.rst
	modified:   source/release-notes.rst

### CODE REVIEW

File: source/release-notes-pxc-v8.0.23.upd.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.. _pdpxc.release_notes_8.0.23:
+
+********************************************************************************
+|pdmysql| 8.0.23 using |PXC| Update
+********************************************************************************
+
+:Date: September 15, 2021
+:Installation: :ref:`installation`
+
+This update of the |PXC|-based variant of the |pdmysql| includes the HAProxy version 2.3.14, which fixes the security vulnerability `CVE-2021-40346 <https://security-tracker.debian.org/tracker/CVE-2021-40346>`_.
```
Comment (LenzGr) [inline source/release-notes-pxc-v8.0.23.upd.rst:10]:
```suggestion
This update of the |PXC|-based variant of the |pdmysql| includes the HAProxy version 2.3.14, which fixes a security vulnerability - see `CVE-2021-40346 <https://security-tracker.debian.org/tracker/CVE-2021-40346>`_ for details.
```

Comment (nastena1606) [reply source/release-notes-pxc-v8.0.23.upd.rst:10]: But it fixes a specific CVE, so it should be "the" here.

Comment (LenzGr) [reply source/release-notes-pxc-v8.0.23.upd.rst:10]: Good point, works for me :)

**Review by LenzGr — APPROVED** [review]
LGTM, thank you!

**Review by ghost — APPROVED** [review]

**Pull request merged**